var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -70,7 +82,10 @@ class Babelstream(CMakePackage, CudaPackage, ROCmPackage):
     variant("mem", values=str, default="DEFAULT", description="Enable MEM Target for CUDA")
     # Raja Conflict
     variant(
-        "offload", values=str, default="none", description="Enable RAJA Target [CPU or NVIDIA]"
+        "offload",
+        values=str,
+        default="none",
+        description="Enable RAJA Target [CPU or NVIDIA] / Offload with custom settings for OpenMP",
```

Comment:
Is the offload still NVIDIA specific? Is `[CPU or NVIDIA]` specifying the valid values?

Reply:
Yes, from the CMake file it was : 

https://github.com/UoB-HPC/BabelStream/blob/2f00dfb7f8b7cfe8c53d20d5c770bccbf8673440/src/raja/model.cmake#L19